rd-kt/rd-gen/src/main/kotlin/com/jetbrains/rd/generator/nova/kotlin/Kotlin11Generator.kt

```diff
@@ -152,6 +153,8 @@ open class Kotlin11Generator(
     protected open fun Member.Reactive.intfSubstitutedMapName(scope: Declaration) =
         "IPerContextMap<${context!!.type.substitutedName(scope)}, " + intfSubstitutedName(scope) + ">"
 
+    private fun Member.getAttrsStr(eolKind: String) = this.getSetting(Attributes)?.toList()?.fold("") { acc, attr -> "$acc@$attr$eolKind" }
```

Comment:
Looks complex. Wouldn't it be the same as the following?
```suggestion
    private fun Member.getAttrsStr(eolKind: String) = this.getSetting(Attributes)?.joinToString { "@$it$eolKind" }
```

Reply:
Not completely. It is the original code that was be shifted to function. And it had $eolKind in the end. But maybe `apply {}` will be a good decision.

Reply:
What's the difference between this and the original, though?

Reply:
sorry, this is carelessness, will fix it